lae_automation/containers.py

```diff
@@ -102,7 +102,7 @@ def create_configuration(deploy_config, details):
         bucket_name=details.bucketname,
         publichost=public_host,
         privatehost=private_host,
-        introducer_furl=details.oldsecrets["internal_introducer_furl"],
+        introducer_furl=details.external_introducer_furl,
```

Comment:
`oldsecrets` is just a plain old dictionary of stuff that gets generated at some point and then is pretty much inert forever after.  It's called `oldsecrets` because (a) that's what the original S4 code called it sometimes and (b) it's now also a legacy format that's never shipped out of the management system.

Contrast this with `external_introducer_furl` which is a property (see model.py) that produces a value taking into account certain other values.

I think the #511 regression was caused by a disagreement between these two expressions.  The former ended up with a garbage port number because the `oldsecrets` structure generation moved from _after_ port assignment to _before_ port assignment (unintentionally).

So switching to the property fixes the problem by getting further away from the bad `oldsecrets`.